buildpipeline/DotNet-CoreClr-Trusted-Windows-x86.json

```diff
@@ -294,6 +294,10 @@
     "VsoPassword": {
       "value": null,
       "isSecret": true
+    },
+    "EnforcePGO": {
+      "value": null,
```

Comment:
`"value": ""` is the usual choice for empty strings. I'm not actually sure what `null` will do for non-secrets.